docs/src/community/maintainer.md

```diff
@@ -0,0 +1,10 @@
+# Current Maintainers
+
+| Name            | Role      | GitHub ID                                          |
+|-----------------|-----------|----------------------------------------------------|
+| Toru Komatsu    | Committer | [@utam0k](https://github.com/utam0k)               |
+| Thomas Schubart | Committer | [@Furisto](https://github.com/Furisto)             |
+| Yashodhan       | Committer | [@YJDoc2](https://github.com/YJDoc2)               |
+| Eric Fang       | Committer | [@yihuaf](https://github.com/yihuaf)               |
+| Sascha Grunert  | Committer | [@saschagrunert](https://github.com/saschagrunert) |
+| Jorge Prendes   | Committer | [@jprendes](https://github.com/jprendes)           |
```

Comment:
Do we want to use [`OWNERS`](https://github.com/kubernetes/kubernetes/blob/master/OWNERS) and [`OWNERS_ALIASES`](https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES) instead of the markdown table? This would be required in any case when considering prow in the future.

Reply:
Have we already decided to use prow? If not I would prefer to keep the markdown table and we can switch to an OWNER file when we switch to prow.